### PR TITLE
Added Google Recaptcha in contact page

### LINKED ISSUE #20: Implement Google ReCaptcha on Contact Page
Email should be sent only on successful captcha verification

### PR DESCRIPTION
**Pull request by saikumar221**
Added a Google Recaptcha on the contact page.
Closes #20 
Use this [link](https://www.google.com/recaptcha/admin/create) to register and get a security key for the site.
Replace it in the below code:
`              <div class="g-recaptcha" style="margin-bottom:10px ;" data-sitekey="Your Site Security Key"></div>
`

### CODE REVIEW

**Comment by pradipta** [comment]
Do you need this key to start working on it and testing it? Is it a blocked from our side?
If yes, I'll send you the key today in some time.

**Comment by saikumar221** [comment]
Hello @pradipta 
I have tested it on my end with a security key I generated, it works perfectly. 
To test it from your end generate the key, put it in the placeholder as mentioned above.
The placement, size of the component can be tested by deploying it on the localhost. Free Google reCaptcha's functionality doesn't work on localhost, it must be deployed on to the website with a domain name. It needs a valid domain with a valid host and must not include any protocol, path, port, query, or fragment.

**Comment by first-brick** [comment]
@pradipta @saikumar221 Do I need it with Recaptcha v1 or v2? Or does it not matter?

**Comment by pradipta** [comment]
@first-brick the keys have been generated, don't worry about that. I will paste it in.
@saikumar221 the div placement and all looks good. Not a problem. But I do not see a validation code. I guess I can still send the query without the captcha verification.
That was what I meant with my first question? Do you want me to merge it with the keys so that you can start working on the validation?

**Comment by saikumar221** [comment]
@pradipta  I generated a key and placed it here, This will now work locally on localhost too.
Also Added the validation logic, please check the latest commit.

File: views/index.ejs [hunk 1]
```diff
@@ -271,7 +273,13 @@
                 var email = $("#email").val();
                 var phone = $("#phone").val();
                 var message = $("#message").val();
-                console.log(name);
+                var response = grecaptcha.getResponse();
+                 console.log(name);
+                if(!response) {
+                  document.getElementById('captchamsg').innerHTML="Please Verify the Captcha";
```
Comment (pradipta) [inline views/index.ejs]: Change this line to `alert("Please Verify the Captcha");`

Comment (pradipta) [reply views/index.ejs:279]: Rest, LGTM

**Review by pradipta — CHANGES_REQUESTED** [review]

**Comment by saikumar221** [comment]
@pradipta requested changes have been made.

**Comment by saikumar221** [comment]
@pradipta requested changes are updated.

**Review by pradipta — APPROVED** [review]

**Review by first-brick — APPROVED** [review]

**Pull request merged**